api/types/lock.go

```diff
@@ -313,3 +313,16 @@ func (t LockTarget) Equals(t2 LockTarget) bool {
 		t.BotInstanceID == t2.BotInstanceID &&
 		t.JoinToken == t2.JoinToken
 }
+
+// NewLockFilter is a convenience method that creates an instance of [*LockFilter].
+func NewLockFilter(inForceOnly bool, targets ...LockTarget) *LockFilter {
```

Comment:
It would have helped me to see the use of this convenience function; specifically, why not use `targets ...*LockTarget` and save the conversion? Aren't proto messages typically passed as pointers, as with older generators, you used to get error messages like `assignment copies lock value: *MessageType contains sync.RWMutex`?